core/templates/domain/skill/SkillDifficultyObjectFactorySpec.ts

```diff
@@ -45,5 +45,19 @@ describe('Skill Difficulty object factory', () => {
       };
       expect(skillDifficulty.toBackendDict()).toEqual(skillDifficultyDict);
     });
+
+    it('should set the description of the skill difficulty instance', () => {
+      var skillDifficulty =
+        skillDifficultyObjectFactory.create('1', 'test skill', 0.3);
+      skillDifficulty.setDescription('new test skill');
```

Comment:
Do a baseline verification first. In general, follow this pattern for tests

1. setup [Create skillDifficulty]
2. baseline verification [Check initial value difficultiy]
3. action [Change the difficulty]
4. endline verification [Verify the difficulty changes.]

For more details: https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests

Reply:
Hi @tianqi-wu, please make sure to comment with "Done" after you have addressed a comment. Also, we recommend not "resolving" comments, the reviewer will do so.

Reply:
We have documented this in step 6 here: https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#instructions-for-making-a-code-change

Reply:
Done.